packages/flutter/lib/src/material/banner.dart

```diff
@@ -201,5 +329,52 @@ class MaterialBanner extends StatelessWidget {
         ],
       ),
     );
+
+    // This provides a static banner during the opt-in phase of the transition to MaterialBanner's new API
```

Comment:
I'm not sure if we need an opt-in phase. We could just make this backwards compatible and leave it available to work both ways. :)